### PR TITLE
add audit ebee backstory

### PR DESCRIPTION
**Pull request by xmulligan**

### CODE REVIEW

**Comment by xmulligan** [comment]
@qmonnet how did you make all of the other images to make them consistent?

**Comment by qmonnet** [comment]
> @qmonnet how did you make all of the other images to make them consistent?

As far as I remember:

1. Removed white background, if any
2. Removed margins (`Crop to content` in Gimp)
3. Resize image to have 577 pixels for the longest side (width or heigth, depending on the pictures). 577 is pretty random from my side, just picked the value that was used in several pictures already and seemed OK for full-size while not making the pictures too big.
4. Resize canvas to have the picture square, 577x577 pixels, keeping the picture in the centre.

I wish I was better at Gimp scripting so we could just automate these, but :shrug:.

**Comment by qmonnet** [comment]
Happy to help if necessary

File: README.md [hunk 1]
```diff
@@ -211,6 +211,16 @@ This eBeeDex lists all known eBees and their back story. If you find any more in
        Backstory:
        What do you need a backstory for when you're resting on a hamoc under a gentle breeze, on the warm and sunny beaches of a Caribbean island? There's no place like the seashore to meet with friends and discuss new networking designs (Oh wait, I'm hearing that Skiing eBee disagrees with this statement). Caribbean eBee invites you for some fun and relaxation. Take a seat next to her. Sip your honey-rum-coconut juice. Listen to the lulling sound of the waves. Enjoy.
    </td>
+  <tr>
+   <td> <img src="" alt="audit bee">
```
Comment (xmulligan) [inline README.md:215]:
```suggestion
   <td> <img src="images/audit-bee.png" alt="audit bee">
```

**Comment by qmonnet** [comment]
audit-ebee

File: README.md [hunk 2]
```diff
@@ -211,6 +211,16 @@ This eBeeDex lists all known eBees and their back story. If you find any more in
...
        What do you need a backstory for when you're resting on a hamoc under a gentle breeze, on the warm and sunny beaches of a Caribbean island? There's no place like the seashore to meet with friends and discuss new networking designs (Oh wait, I'm hearing that Skiing eBee disagrees with this statement). Caribbean eBee invites you for some fun and relaxation. Take a seat next to her. Sip your honey-rum-coconut juice. Listen to the lulling sound of the waves. Enjoy.
    </td>
+  <tr>
+   <td> <img src="images/audit-bee.png" alt="audit bee">
+   </td>
+   </td>
+   <td>#23: Audit eBee
+     <br />
+     <br />
+       Backstory:
```
Comment (qmonnet) [inline README.md:221]:
```suggestion
```

**Review by qmonnet — CHANGES_REQUESTED** [review]

**Review by qmonnet — APPROVED** [review]
Looks all good!
I'd suggest hitting squash-and-merge.

**Comment by qmonnet** [comment]
Oh wait I can do that myself

**Pull request merged**